src/views/Home.vue

```diff
@@ -3,10 +3,33 @@
     <h1>This is a 🏡 component.</h1>
     <div class="main">
       <section class="left-section">
-        <h1 class="module">Module 1</h1>
-        <h1 class="module">Module 2</h1>
-        <h1 class="module">Module 3</h1>
-        <h1 class="module">Module 4</h1>
+        <router-link :to="{
+          name: 'Module',
+          params: { id: 1}
+        }">
+          <h1 class="module">Module 1</h1>
+        </router-link>
+
+        <router-link :to="{
+          name: 'Module',
+          params: { id: 2}
+        }">
+          <h1 class="module">Module 2</h1>
+        </router-link>
+
+        <router-link :to="{
+          name: 'Module',
+          params: { id: 3}
+        }">
+          <h1 class="module">Module 3</h1>
+        </router-link>
+
+        <router-link :to="{
+          name: 'Module',
+          params: { id: 4}
+        }">
+          <h1 class="module">Module 4</h1>
+        </router-link>
```

Comment:
I don't know if this is the way to do this, so need to talk it out with Jessica.